doc/EnvironmentVariables.md

```diff
@@ -84,7 +84,10 @@
-| MqttBroker__MaxQueuedBytes              | Maximum number of bytes that can be in the outgoing queue for a single session                                                                     | int32 (arm), int64 (amd64)                                            | 0 (no limit)                         |
-| MqttBroker__WhenFull                    | Defines what to do in case either max queued messages or max queued bytes is reached                                                               | drop_new, drop_old                                                    | drop_new                       |
\ No newline at end of file
+| MqttBroker__MaxInflightMessages         | Maximum number of QoS 1 messages that can be in the process of being sent simultaneously for a single session                                      | int32                                                                 | 16                            |
+| MqttBroker__MaxQueuedMessages           | Maximum number of messages that can be in the outgoing queue for a single session                                                                  | int32                                                                 | 1000                          |
+| MqttBroker__MaxQueuedBytes              | Maximum number of bytes that can be in the outgoing queue for a single session                                                                     | int32 (arm), int64 (amd64)                                            | 0 (no limit)                  |
+| MqttBroker__WhenFull                    | Defines what to do in case either max queued messages or max queued bytes is reached                                                               | drop_new, drop_old                                                    | drop_new                      |
+| MqttBridge__StorageMaxFileSize          | If `UsePersistentStorage` is set, defines maximum size (in bytes) of the MQTT Bridge storage. This storage is separate from edgehub messages / twins storage.   | int64                                                    | 33554432 (32mb)               |
+| MqttBridge__StorageFlushOptions         | If `UsePersistentStorage` is set, defines MQTT Bridge storage flush options.                                                                                    | AfterEachWrite, Off                                      | AfterEachWrite                |
+| MqttBridge__StorageMaxMessages          | If `UsePersistentStorage` is not set, defines max in-memory queued messages in the MQTT Bridge.                                                                 | int32 (arm), int64 (amd64)                               | 1000                          |
```

Comment:
Same for this.

Reply:
@varunpuranik, I have renamed the vars. I used the name `MqttBridge__StorageMaxMemoryMessages` without words "Usage" or "Size" in it, for memory option, to emphasize that this is __a number of messages__, and not bytes.